tests/locals.tf

```diff
@@ -0,0 +1,9 @@
+locals {
+  # Name, name tag, log name related locals
+  date_ymd      = "${substr(data.null_data_source.start_time.inputs.timestamp,0,4)}${substr(data.null_data_source.start_time.inputs.timestamp,5,2)}${substr(data.null_data_source.start_time.inputs.timestamp,8,2)}" #equivalent of $(date +'%Y%m%d')
+  date_hm       = "${substr(data.null_data_source.start_time.inputs.timestamp,11,2)}${substr(data.null_data_source.start_time.inputs.timestamp,14,2)}"                                                               #equivalent of $(date +'%H%M')
+  full_build_id = "${var.codebuild_id == "" ? format("notcb:%s", uuid()) : var.codebuild_id}"                                                                                                                        #128-bit rfc 4122 v4 UUID
+  build_id      = "${substr(element(split(":",local.full_build_id),1), 0, 8)}${substr(element(split(":",local.full_build_id),1), 9, 4)}"                                                                             #extract node portion of uuid (last 6 octets) for brevity
+  test_suffix   = "tf-wam-test-${random_id.this.hex}"
```

Comment:
I think we can use `build_id` instead of the `random_id`. This is just used to link the names, so naming them for the build_id would be preferable.

Reply:
rgr, updated